source/Halibut.Tests/Diagnostics/UnpackFromContainersFixture.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Reflection;
+using FluentAssertions;
+using Halibut.Diagnostics;
+using NUnit.Framework;
+
+namespace Halibut.Tests.Diagnostics;
+
+public class UnpackFromContainersFixture
```

Comment:
I didn't understand what this was doing so I wrote a test. Which I have accidentally committed. Would you like this to come along for the ride? or should I drop this test?